app/views/comments/_sort_option.html.erb

```diff
@@ -0,0 +1,15 @@
+<li class="comment-sort-option">
+  <a
+    href="<%= @article.path %>?comments_sort=<%= sort_order %>"
```

Comment:
I'm not sure if this was discussed at some point, but after selecting a filter the page is refreshed and users have to scroll back down to the comments section. Maybe we can add a simple anchor to the link to avoid this, what do you think?

```suggestion
    href="<%= @article.path %>?comments_sort=<%= sort_order %>#comments"
```

Reply:
Sure.  Any idea about error I posted https://github.com/forem/forem/pull/18349#issuecomment-1218378312

Reply:
So I've seen that before a few times but only in development mode (locally), but it seems like it happens when to try to access a post and the post itself is somewhat bugged. I believe this has happened after switching branches a few times and something has changed/modified over time.

My suggestion would be to merge main so your branch is up to date and then recreate the DB, hopefully you won't run into this anymore. Something along the lines of:

```ruby
# Drop the DB
bundle exec rails db:drop

# Recreate & seed the DB
bin/setup

#Note: Only if the local DB isn't seeded you can try to seed it manually with `bundle exec rails db:seed`

# Startup again
bin/startup
```

Let me know if this solves the issue you're seeing

Reply:
seems to resolved, thank you

Reply:
@fdocr Opened for Review!!! Excited